### PR TITLE
[zh-cn]  Localize /en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

### PR DESCRIPTION
**Pull request by ktKongTong**
[hi] Localize /en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md
fixes https://github.com/kubernetes/website/issues/38640

### CODE REVIEW

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+了解详细信息，请参阅驱动程序的文档。
+
+## {{% heading "whatsnext" %}}
+
+<!-- 
+ - For more information on the design, see the
+[Dynamic Resource Allocation KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3063-dynamic-resource-allocation/README.md).
+ -->
+- 了解更多该设计的信息，
+参阅[动态资源分配 KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3063-dynamic-resource-allocation/README.md)。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:337]:
```suggestion
  参阅[动态资源分配 KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3063-dynamic-resource-allocation/README.md)。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 2]
```diff
@@ -0,0 +1,337 @@
+---
+title: 动态资源分配
+content_type: concept
+weight: 65
+---
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:5]:
```suggestion
---
title: 动态资源分配
content_type: concept
weight: 65
---
<!--
reviewers:
- klueska
- pohly
title: Dynamic Resource Allocation
content_type: concept
weight: 65
-->
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+<!-- 
+Dynamic resource allocation is a new API for requesting and sharing resources
+between pods and containers inside a pod. It is a generalization of the
+persistent volumes API for generic resources. Third-party resource drivers are
+responsible for tracking and allocating resources. Different kinds of
+resources support arbitrary parameters for defining requirements and
+initialization.
+ -->
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:18]:
```suggestion
-->
```
This prefixed spaces can be removed in this page.

**Comment by tengqm** [comment]
pls squash your PR into a single commit.

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+<!-- 
+In contrast to native resources (CPU, RAM) and extended resources (managed by a
+device plugin, advertised by kubelet), the scheduler has no knowledge of what
+dynamic resources are available in a cluster or how they could be split up to
+satisfy the requirements of a specific ResourceClaim. Resource drivers are
+responsible for that. They mark ResourceClaims as "allocated" once resources
+for it are reserved. This also then tells the scheduler where in the cluster a
+ResourceClaim is available.
+-->
+与本地资源（CPU、RAM）和扩展资源（由设备插件管理，由 kubelet 公布）不同，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:201]:
```suggestion
与原生资源（CPU、RAM）和扩展资源（由设备插件管理，并由 kubelet 公布）不同，
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 5]
```diff
@@ -0,0 +1,345 @@
...
+调度器不知道集群中有哪些动态资源，
+也不知道如何将它们拆分以满足特定 ResourceClaim 的要求。
+资源驱动程序负责这些任务。
+资源驱动程序在为 ResourceClaim 保留资源后将其标记为“已分配”。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:205]:
```suggestion
资源驱动程序在为 ResourceClaim 保留资源后将其标记为“已分配（Allocated）”。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 6]
```diff
@@ -0,0 +1,345 @@
...
+然后告诉调度器集群中可用的 ResourceClaim 的位置。
+
+<!-- 
+ResourceClaims can get allocated as soon as they are created ("immediate
+allocation"), without considering which Pods will use them. The default is to
+delay allocation until a Pod gets scheduled which needs the ResourceClaim
+(i.e. "wait for first consumer").
+-->
+ResourceClaim 可以在创建时就进行分配（“立即分配”），不用考虑哪些 Pod 将使用它。
+默认情况下将延迟分配，直到需要 ResourceClaim 的 Pod 被调度时
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:215]:
```suggestion
默认情况下采用延迟分配，直到需要 ResourceClaim 的 Pod 被调度时
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 7]
```diff
@@ -0,0 +1,345 @@
...
+然后，资源驱动程序为分配其 ResourceClaim，以便资源可用于该节点。
+完成后，Pod 就会被调度。
+
+<!-- 
+As part of this process, ResourceClaims also get reserved for the
+Pod. Currently ResourceClaims can either be used exclusively by a single Pod or
+an unlimited number of Pods.
+-->
+作为此过程的一部分，ResourceClaim 会为 Pod 保留。
+目前，ResourceClaim 可以由单个 Pod 独占使用或不限数量的 Pod 使用。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:242]:
```suggestion
目前，ResourceClaim 可以由单个 Pod 独占使用或不限数量的多个 Pod 使用。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 8]
```diff
@@ -0,0 +1,345 @@
...
+`DynamicResourceAllocation` [feature
+gate](/docs/reference/command-line-tools-reference/feature-gates/) and the
+`resource.k8s.io/v1alpha1` {{< glossary_tooltip text="API group"
+term_id="api-group" >}} are enabled. For details on that, see the
+`--feature-gates` and `--runtime-config` [kube-apiserver
+parameters](/docs/reference/command-line-tools-reference/kube-apiserver/).
+kube-scheduler, kube-controller-manager and kubelet also need the feature gate.
+-->
+动态资源分配是一个 **alpha 特性**，只有在启用 `DynamicResourceAllocation`
+[特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:289]:
```suggestion
[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 9]
```diff
@@ -0,0 +1,345 @@
...
+和 `resource.k8s.io/v1alpha1` {{< glossary_tooltip text="API 组"
+term_id="api-group" >}} 时才启用。
+有关详细信息，参阅 `--feature-gates` 和 `--runtime-config`
+[kube-apiserver 参数](/zh/docs/reference/command-line-tools-reference/kube-apiserver/)。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:293]:
```suggestion
[kube-apiserver 参数](/zh-cn/docs/reference/command-line-tools-reference/kube-apiserver/)。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 10]
```diff
@@ -0,0 +1,345 @@
...
+kube-scheduler、kube-controller-manager 和 kubelet 也需要特性门控。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:294]:
```suggestion
kube-scheduler、kube-controller-manager 和 kubelet 也需要设置该特性门控。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 11]
```diff
@@ -0,0 +1,345 @@
...
+error: the server doesn't have a resource type "resourceclasses"
+```
+
+<!-- 
+The default configuration of kube-scheduler enables the "DynamicResources"
+plugin if and only if the feature gate is enabled. Custom configurations may
+have to be modified to include it.
+-->
+kube-scheduler 的默认配置仅在启用特性门控时才启用 "DynamicResources" 插件。
+自定义配置可能需要修改才能启用它。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:329]:
```suggestion
自定义配置可能需要被修改才能启用它。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 12]
```diff
@@ -0,0 +1,345 @@
...
+
+<!-- 
+In addition to enabling the feature in the cluster, a resource driver also has to
+be installed. Please refer to the driver's documentation for details.
+-->
+除了在集群中启用该功能外，还必须安装资源驱动程序。
+了解详细信息，请参阅驱动程序的文档。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:336]:
```suggestion
欲了解详细信息，请参阅驱动程序的文档。
```

**Comment by ktKongTong** [comment]
> pls squash your PR into a single commit.

done

**Pull request merged**